lab_5_scrapper/scrapper.py

```diff
@@ -13,70 +73,130 @@ def __init__(self, path_to_config: Path) -> None:
+
+        for element in self._seed_urls:
+            if not isinstance(element, str):
+                raise IncorrectSeedURLError
+
+            if 'https://' not in element:
+            #if not re.match(r'https://', element):
+                raise IncorrectSeedURLError
+
+        if not isinstance(self._num_articles, int)\
+                or isinstance(self._num_articles, bool) or self._num_articles <= 0:
+            raise IncorrectNumberOfArticlesError("Total number of articles to parse is not integer")
+
+        if self._num_articles < 1 \
+                or self._num_articles > 150:
+            raise NumberOfArticlesOutOfRangeError("Total number of articles is out of range from 1 to 150")
+
+        if not isinstance(self._headers, dict):
+            raise IncorrectHeadersError("Headers are not in a form of dictionary")
+
+        if not isinstance(self._encoding, str):
+            raise IncorrectEncodingError("Encoding must be specified as a string")
+
+        if not isinstance(self._timeout, int):
+            raise IncorrectTimeoutError
+
+        if self._timeout <= 0 or self._timeout > 60:
+            raise IncorrectTimeoutError("Timeout value must be a positive integer less than 60")
+
+        if not isinstance(self._headless_mode, bool):
+            raise IncorrectVerifyError
+
+        if not isinstance(self._should_verify_certificate, bool):
+            raise IncorrectVerifyError("Verify certificate value must either be True or False")
 
     def get_seed_urls(self) -> list[str]:
         """
         Retrieve seed urls
         """
-        pass
+        return self.config_obj.seed_urls
 
     def get_num_articles(self) -> int:
         """
         Retrieve total number of articles to scrape
         """
-        pass
+        return self.config_obj.total_articles
 
     def get_headers(self) -> dict[str, str]:
         """
         Retrieve headers to use during requesting
         """
-        pass
+        return self.config_obj.headers
 
     def get_encoding(self) -> str:
         """
         Retrieve encoding to use during parsing
         """
-        pass
+        return self.config_obj.encoding
 
     def get_timeout(self) -> int:
         """
         Retrieve number of seconds to wait for response
         """
-        pass
+        return self.config_obj.timeout
 
     def get_verify_certificate(self) -> bool:
         """
         Retrieve whether to verify certificate
         """
-        pass
+        return self.config_obj.should_verify_certificate
 
     def get_headless_mode(self) -> bool:
         """
         Retrieve whether to use headless mode
         """
-        pass
+        return self.config_obj.headless_mode
 
 
 def make_request(url: str, config: Config) -> requests.models.Response:
     """
     Delivers a response from a request
     with given configuration
     """
-    pass
+
+    response = requests.get(url, config.get_headers())
+    time.sleep(config.get_timeout())
```

Comment:
config's timeout is for get request, for sleeps you need to generate some random value from interval